local-cli/generator-common/index.js

```diff
@@ -1,16 +1,35 @@
+// @ts-check
+
 const fs = require('fs');
-const chalk = require('chalk');
+const chalk = require('chalk').default;
```

Comment:
Off-hand, need to check in more detail, I would think that “Apple PR” CI task is failing because of this change.

Reply:
I checked out the change locally and hit the same error as in the CI test.   I changed it back to `const chalk = require('chalk');` in both this file and `local-cli/generator-macos/index.js` and it worked.

Reply:
Unsure how this built for me locally, but probably just a matter of some build artefacts lingering around. I figured out that this version of `chalk` actually comes with its own TS typings and so the trick was to ensure vscode would _not_ pull typings from DT.